src/frontend/list/listItem.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
very nitpicky, but file names should use `kebab-case`, so this should be `list-item.js` (it's [in the PHP code standards,](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions) not explicitly in the JS standards, but I think it inherits).